docs/en/configuration.md

```diff
@@ -140,6 +153,9 @@ class FormPage extends Page
 }
 ```
 
+**Note:** Editing a non-default locale affects the `SiteTree(_live)` table. In contrast to SilverStripe 3, the 
```

Comment:
It doesn't matter if the locale is default or not. In fluent 4 vs 3, the "default" locale matters a lot less than it once did as far as content authoring is going.

Now all it means is that the default will be the first locale a user sees on the frontend.

Reply:
Okay, I am going to change the phrasing slighlty. Let me know if that sounds better.